umbraco/Umbraco-CMS
# V8 Plugins: Changed treeService.getTreePackageFolder to use localeCompare

### Prerequisites


Fix for issue [#5069](https://github.com/umbraco/Umbraco-CMS/issues/5069)

### Description

As per description in issue ^^ I changed treeService.getTreePackageFolder to use localeCompare
instead of the '===' comparison. 

Steps to test:
- Create a Custom tree controller
- Add a PluginController attribute
- Override the required methods
- Override CreateRootNode
- Add a route.RoutePath with a different casing then used in the PluginController attribute
- Create the required Plugin folders.

The correct path should now be found when opening the tree in the Umbraco backoffice.

Note, if anyone knows a better solution then adding '1' to the localeCompare result feedback is welcome.
